crates/pop-cli/src/commands/call/parachain.rs

```diff
@@ -639,6 +692,29 @@ mod tests {
 		cli.verify()
 	}
 
+	#[tokio::test]
+	async fn user_cancel_submit_extrinsic_from_call_data_works() -> Result<()> {
+		let client = set_up_client("wss://rpc1.paseo.popnetwork.xyz").await?;
+		let call_config = CallParachainCommand {
```

Comment:
For future, deriving Default on the command might make all this initialisation much simpler via `CallParachainCommand { ..Default::default() }`